arbor/morph/place_pwlin.cpp

```diff
@@ -220,9 +220,7 @@ std::pair<mlocation, double> place_pwlin::closest(double x, double y, double z)
             // v and w are the proximal and distal ends of the segment.
             const p3d v = seg.prox;
             const p3d w = seg.dist;
-
             const p3d vw = w-v;
-            const p3d vp = p-v;
```

Comment:
Strange to remove one but not both. Any idea why?

Reply:
`vp` is constructed inside a conditional and closer to its sole site of use. So, it's clearer and it might save the odd cycle.